flexmeasures/ui/templates/crud/asset.html

```diff
@@ -166,6 +166,23 @@ <h3>Edit {{ asset.name }}</h3>
                                 dif + pad(Math.floor(Math.abs(tzo) / 60)) +
                                 ':' + pad(Math.abs(tzo) % 60);
                         }
+
+                        $(window).ready(() => {
+                            picker.on('selected', (startDate, endDate) => {
+                                startDate = encodeURIComponent(toIsoString(startDate.toJSDate()));
+                                // add 1 day to end date as datepicker does not include the end date day
+                                endDate = endDate.toJSDate();
+                                endDate.setDate(endDate.getDate() + 1);
```

Comment:
Possible source of a bug. I just tested this, but when I select 2 days and then refresh, the url indicates a period of 3 days. And when I refresh, it selects those 3 days. (When I refresh again, it does remain at 3 days.)